apps/roam/src/components/CreateNodeDialog.tsx

```diff
@@ -71,7 +86,16 @@ const CreateNodeDialog = ({
       // the correct reference. The reference format should be determined by the
       // node's specification.
       const pageRef = `[[${formattedTitle}]]`;
-      await updateBlock({ uid: sourceBlockUid, text: pageRef });
+      const newBlockUid = await createBlock({
```

Comment:
Why do we need to createBlock instead of updateBlock?  Ideally we can keep using updateBlock which will keep the create date/user data.

Reply:
Don't we have to move the existing block and its decendents as child of the new dg node block?

<img width="680" height="362" alt="image" src="https://github.com/user-attachments/assets/470dc19b-aee4-4a19-9bdd-25ce18c09fcb" />